src/DynamoInstall/Release.xsl

```diff
@@ -55,10 +55,6 @@
     <xsl:key name="nunit-search" match="wix:Component[contains(wix:File/@Source, 'nunit')]" use="@Id"/>
     <xsl:template match="wix:Component[key('nunit-search', @Id)]" />
 	
-    <!--Exclude OpenSans* fonts-->
-    <xsl:key name="OpenSans-search" match="wix:Component[contains(wix:File/@Source, 'OpenSans')]" use="@Id"/>
-    <xsl:template match="wix:Component[key('OpenSans-search', @Id)]" />
-  
```

Comment:
Have we ensured the original intention of this removal? I just want to be very certain about us not reintroduce this by mistake :)